### PR TITLE
Moving shared adapter functionality into HBaseRequestAdapter.

### PR DESCRIPTION
**Pull request by sduskis**
BigtableTable and BatchExecutor did similar things.  Moved the shared functionality into BatchExecutor.

### CODE REVIEW

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import org.apache.hadoop.hbase.client.Scan;
+
+import com.google.bigtable.v1.MutateRowRequest;
+import com.google.bigtable.v1.ReadModifyWriteRowRequest;
+import com.google.bigtable.v1.ReadRowsRequest;
+import com.google.cloud.bigtable.grpc.BigtableClusterName;
+import com.google.cloud.bigtable.grpc.BigtableTableName;
+
+ /**
+ * Adapts HBase Deletes, Gets, Puts, RowMutations, Appends, Increments to Bigtable requests
```
Comment (kevinsi4508) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java]: Adapts HBase Deletes, Gets, Puts, RowMutations, Appends, and Increments to Bigtable requests.

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    return builder.build();
+  }
+
+  public MutateRowRequest adapt(org.apache.hadoop.hbase.client.Mutation mutation) {
+    MutateRowRequest.Builder builder = mutationAdapter.adapt(mutation);
+    builder.setTableName(bigtableTableName.toString());
+    return builder.build();
+  }
+
+  public Result adapt(com.google.bigtable.v1.Row row) {
```
Comment (kevinsi4508) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java]: Adapting row is for response not request as stated in the document.

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java [hunk 3]
```diff
@@ -405,4 +370,12 @@ public void batch(List<? extends Row> actions, @Nullable Object[] results)
     }
     return exists;
   }
+
+  public BigtableDataClient getClient() {
+    return client;
+  }
+
+  public HBaseRequestAdapter getHbaseAadapter() {
```
Comment (kevinsi4508) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java]: getHbaseAdapter

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java (hunk 2 above)
Comment (sduskis) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java]: I'll revert this one.

**Comment by sduskis** [comment]
Addressed the comments.  PTAL

**Comment by kevinsi4508** [comment]
LGTM, please resolve the conflicts first.

**Pull request merged**